src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -716,8 +716,8 @@ public function getRequestedTimes()
 
     private function formatTimeHuman($time)
     {
-        $stime = gmdate(' H:i', $time['start_time'] + date('Z'));
-        $etime = gmdate('H:i', $time['start_time'] + $time['duration'] + date('Z'));
+        $stime = date(' H:i', $time['start_time']);
+        $etime = date('H:i', $time['start_time'] + $time['duration']);
```

Comment:
I guess this is actually the same thing. I think I may have only done it like that to make it clear the localisation adjustment was intended.